aten/src/ATen/native/ReduceOpsUtils.h

```diff
@@ -212,7 +212,7 @@ static TensorIterator make_reduction(
 
 static TensorIterator make_reduction(
     const char* name, Tensor& result1, Tensor& result2, const Tensor& self, IntArrayRef dim,
-    bool keepdim, ScalarType dtype1, ScalarType dtype2, bool promote_gpu_output_dtypes=true)
```

Comment:
Do you know the historical context for this argument?

Reply:
Yes. I added a brief reference for it in the PR note. I reviewed with @ngimel and the team at NVIDIA before removing it.